src/background/stats.js

```diff
@@ -0,0 +1,56 @@
+/**
+ * Ghostery Browser Extension
+ * https://www.ghostery.com/
+ *
+ * Copyright 2017-present Ghostery GmbH. All rights reserved.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0
+ */
+import AnonymousCommunication from '@whotracksme/webextension-packages/packages/anonymous-communication';
+import Storage from '@whotracksme/webextension-packages/packages/anonymous-communication/storage';
+import {
+  getTimeAsYYYYMMDDHH,
+  getTimeAsYYYYMMDD,
+  getTrustedUtcTime,
+} from '@whotracksme/webextension-packages/packages/anonymous-communication/timestamps';
+
+const HPN_URL = 'https://collector-hpn.ghostery.net';
+const CONFIG_URL = 'https://api.ghostery.net/api/v1/config';
+
+async function stats() {
+  const storage = new Storage('hpn-public-keys');
+  await storage.init();
+
+  const location = await fetch(CONFIG_URL)
```

Comment:
Would recommend to handle errors. A typical patterns looks like that:

```
const response = await fetch(...);
if (!response.ok) {
  throw new Error(`Failed to send data (${response.statusText})`);
}
```

Otherwise, you will try to parse responses for non 2xx requests, which leads to hard to read error messages.